### PR TITLE
Make passing org to flow validation endpoint optional 

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: web/flow/flow.go [hunk 1]
```diff
@@ -95,27 +98,32 @@ func handleValidate(ctx context.Context, s *web.Server, r *http.Request) (interf
...
-	if err != nil {
-		return nil, http.StatusBadRequest, err
-	}
-
-	// build our session assets
-	sa, err := models.NewSessionAssets(org)
-	if err != nil {
-		return nil, http.StatusInternalServerError, errors.Wrapf(err, "unable get session assets")
+	// if we have an org ID, build a session assets for it
+	if request.OrgID != 0 {
```
Comment (nicpottier) [inline web/flow/flow.go:117]: `models.NilOrgID`

File: web/flow/flow.go [hunk 2]
```diff
@@ -95,27 +98,32 @@ func handleValidate(ctx context.Context, s *web.Server, r *http.Request) (interf
...
+		org, err := models.NewOrgAssets(s.CTX, s.DB, request.OrgID, nil)
+		if err != nil {
+			return nil, http.StatusBadRequest, err
+		}
+
+		sa, err = models.NewSessionAssets(org)
```
Comment (nicpottier) [inline web/flow/flow.go:123]: Shouldn't this be out of the if?

Comment (nicpottier) [reply web/flow/flow.go:123]: Oh NM

File: web/flow/flow.go (hunk 1 above)
Comment (rowanseymour) [reply web/flow/flow.go:117]: updated

**Pull request merged**